### PR TITLE
Homer 385 case study landing ja

### PR DESCRIPTION
**Pull request by jaube05**

### CODE REVIEW

File: src/pages/case-studies.js [hunk 1]
```diff
@@ -1,38 +1,137 @@
...
       }
     }
+  };
+
+  let cases;
+  const caseStudies = data.allStrapiCaseStudy.nodes;
+
+  // Handle filter states
+  if (filterState) {
+        // Todo: Only handles one input filter at a time, not multiple
```
Comment (jaube05) [inline src/pages/case-studies.js:80]: @iris-i I can filter the data based on the input filters but only one at a time. Ive tried everything to get it working to accept multiple checks.

Comment (iris-i) [reply src/pages/case-studies.js]: @jaube05 I made some changes to make this accept multiple filters. Lmk if it works as expected.

Comment (jaube05) [reply src/pages/case-studies.js]: yes it worked thank you

File: src/pages/case-studies.js [hunk 2]
```diff
@@ -1,38 +1,261 @@
...
+        <title data-react-helmet='true'>Case Studies Landing page</title>
+      </Helmet>
+      <Hero
+        title='Work that makes a difference'
+        description='Our work impacts the daily lives of millions of people. See how we’ve helped agencies build resilient services at scale.'
+      />
+      <section className='section--case-studies--filter'>
+        <h2 className='body'>Filter by service category</h2>
+        <div className={`checkboxes ${filterState}`}>
+          {categories.map((item, index) => (
```
Comment (iris-i) [inline src/pages/case-studies.js]: When mapping through an array, you don't need to pass the index to get each item in the array. `categories[index].id` can be replaced by `item.id`.

Also, when mapping through an array/object, react needs a `key` attribute. More info here: https://reactjs.org/docs/lists-and-keys.html

**Review by iris-i — APPROVED** [review]

**Pull request merged**